sass/components/_dialog.scss

```diff
@@ -98,6 +98,7 @@
                 padding: 4px;
                 position: relative;
                 top: -6px;
+                right: -2.5px;
```

Comment:
and here removee both `top` and `right` positioning

Reply:
@mahdiyeh-fs, Here similar situation: if I remove them, the design will be changed. For a block with class 'sc-dialog__head' will be adding additional indent, which wasn’t there before fixing the bug.